test/parallel/test-os.js

```diff
@@ -259,3 +259,10 @@ if (!common.isIBMi) {
 
 is.number(+os.freemem, 'freemem');
 is.number(os.freemem(), 'freemem');
+
+const devNull = os.devNull;
+if (common.isWindows) {
+  assert.strictEqual(devNull, '\\\\.\\nul');
+} else {
+  assert.strictEqual(devNull, '/dev/null');
+}
```

Comment:
Any reason not to store `os.devNull` in a separate variables? I'm not getting it.
```suggestion
if (common.isWindows) {
  assert.strictEqual(os.devNull, '\\\\.\\nul');
} else {
  assert.strictEqual(os.devNull, '/dev/null');
}
```

Reply:
Consistency with https://github.com/nodejs/node/blob/1fc60abbcdc774fa044f6abb35fce9b8b9d48ad8/test/parallel/test-os.js#L186-L191